src/components/darien/DarienHome.jsx

```diff
@@ -17,47 +19,54 @@ import {
 
-class DarienHome extends React.Component {
-  constructor(props) {
-    super(props);
-  }
+const DarienHome = (props) => {
+  const signedIn = (props.user && props.initialised);
+  const selectedProgramExists = (props.programsStatus === PROGRAMS_STATUS.SUCCESS && props.selectedProgram);
+  const name = (selectedProgramExists && props.selectedProgram.name) ? props.selectedProgram.name : '';
 
-  render() {
-    const selectedProgramExists = (this.props.programsStatus === PROGRAMS_STATUS.SUCCESS && this.props.selectedProgram);
-    const name = (selectedProgramExists && this.props.selectedProgram.name) ? this.props.selectedProgram.name : '';
-
-    return (
-      <ProgramHome className="darien-home">
-        <Hero
-          className="program-home__hero"
-          background={<Image src="https://placeimg.com/1000/1000/nature/any" fit="cover" />}
-          backgroundColorIndex="dark"
-          size="medium"
-        >
-          <Box align="center"><Heading className="program-home__header">{name}</Heading></Box>
-        </Hero>
-        <Section
-          className="program-home__section"
-          align="center"
-          colorIndex="accent-3"
-          direction="column"
-          margin={{ vertical: 'none', horizontal: 'none' }}
-          pad={{ vertical: 'none', horizontal: 'none' }}
-          justify="center"
-        >
-          <Box align="center" direction="column">
-            <Paragraph>Investigate ecological questions by exploring trail camera data using an interactive map. Filter and download data to perform analyses and test hypotheses. If you are an educator, you can set up private classrooms and invite your students to join. Curate data sets or let your students explore on their own. Guided activities and supporting educational resources are also available. If you are a student or you simply want to explore the data, click the Explorer button.</Paragraph>
-            <Paragraph>Are you an educator or a student/explorer? Make your selection to get started!</Paragraph>
+  return (
+    <ProgramHome className="darien-home">
+      <Hero
+        className="program-home__hero"
+        background={<Image src="https://placeimg.com/1000/1000/nature/any" fit="cover" />}
+        backgroundColorIndex="dark"
+        size="medium"
+      >
+        <Box align="center"><Heading className="program-home__header">{name}</Heading></Box>
+      </Hero>
+      <Section
+        className="program-home__section"
+        align="center"
+        colorIndex="accent-3"
+        direction="column"
+        margin={{ vertical: 'none', horizontal: 'none' }}
+        pad={{ vertical: 'none', horizontal: 'none' }}
+        justify="center"
+      >
+        <Box align="center" direction="column">
+          <Paragraph>Investigate ecological questions by exploring trail camera data using an interactive map. Filter and download data to perform analyses and test hypotheses.</Paragraph>
+          
+          <Box pad="medium" size="medium">
```

Comment:
Just a minor styling comment, but perhaps the Box should be defined as a bigger size to allow for the paragraphs to be displaying inline? Even with a fairly high resolution screen, I still have to scroll to see both options for the lab to get started.